packages/evolution-common/src/utils/__tests__/ParamsValidatorUtils.test.ts

```diff
@@ -0,0 +1,606 @@
+            expect(errors).toHaveLength(1);
+            expect(errors[0].message).toContain('should be an array of valid uuids');
+        });
+
+        test('should return an error for an array with invalid UUIDs', () => {
+            const errors = ParamsValidatorUtils.isArrayOfUuids('attr', ['valid-uuid', 'invalid'], 'TestClass');
+            expect(errors).toHaveLength(1);
+            expect(errors[0].message).toContain('should be an array of valid uuids');
+        });
+    });
+
+    describe('isArrayOf', () => {
+        test('should return no errors for an array of instances of the specified class', () => {
+            const errors = ParamsValidatorUtils.isArrayOf('attr', [new TestClass(), new TestClass()], 'TestClass', TestClass);
+            expect(errors).toEqual([]);
+        });
+
+        test('should return no errors for an undefined value', () => {
+            const errors = ParamsValidatorUtils.isArrayOf('attr', undefined, 'TestClass', TestClass);
+            expect(errors).toEqual([]);
+        });
+
+        test('should return no errors for an empty array', () => {
+            const errors = ParamsValidatorUtils.isArrayOf('attr', [], 'TestClass', TestClass);
+            expect(errors).toEqual([]);
+        });
+
+        test('should return an error for a non-array value', () => {
+            const errors = ParamsValidatorUtils.isArrayOf('attr', 'invalid', 'TestClass', TestClass);
+            expect(errors).toHaveLength(1);
+            expect(errors[0].message).toContain('should be an array of TestClass');
+        });
+
+        test('should return an error for an array with non-instances of the specified class', () => {
+            const errors = ParamsValidatorUtils.isArrayOf('attr', [new TestClass(), 'invalid'], 'TestClass', TestClass);
+            expect(errors).toHaveLength(1);
+            expect(errors[0].message).toContain('should be an array of TestClass');
+        });
+    });
+
+    describe('isGeojsonPoint', () => {
+        test('should return no errors for a valid GeoJSON Point', () => {
+            const point: GeoJSON.Feature<GeoJSON.Point> = {
+                type: 'Feature',
+                properties: {},
+                geometry: {
+                    type: 'Point',
+                    coordinates: [0, 0]
+                }
+            };
+            const errors = ParamsValidatorUtils.isGeojsonPoint('attr', point, 'TestClass');
+            expect(errors).toEqual([]);
+        });
+
+        test('should return no errors for an undefined value', () => {
+            const errors = ParamsValidatorUtils.isGeojsonPoint('attr', undefined, 'TestClass');
+            expect(errors).toEqual([]);
+        });
+
+        test('should return an error for an invalid GeoJSON Point', () => {
+            const errors = ParamsValidatorUtils.isGeojsonPoint('attr', { type: 'Feature', geometry: { type: 'Invalid' } }, 'TestClass');
```

Comment:
ça pourrait être un type: 'point', mais sans coordonnées. Ou alors n'importe quel autre vrai type geojson. Aussi quid d'un featureCollection avec un seul feature de type point?

Reply:
added test for [] and undefined coordinates, but I think it is the job if isFeature and isPoint/isMulti... to test these cases.